casanovo/denovo/model_runner.py

```diff
@@ -255,7 +262,36 @@ def initialize_trainer(self, train: bool) -> None:
                 strategy=self._get_strategy(),
                 val_check_interval=self.config.val_check_interval,
                 check_val_every_n_epoch=None,
+                log_every_n_steps=1,
             )
+
+            if self.config._user_config.get("log_metrics"):
```

Comment:
**question:** Why are you using `_user_config`? As indicated by the underscore, this is supposed to be private. Instead, it should just be `self.config["log_metrics"]`.

Reply:
When first implemented this I used `self.config["log_metrics"]`, but I ran into an issue where the default value was being retrieved instead of the actual value in the config file. I'm not running into this issue now though so I'll go ahead and use the config objects item retrieval instead.